models/article-models.js

```diff
@@ -1,7 +1,17 @@
 const db = require('../db/connection');
 
 exports.selectArticleById = (id) => {
-    return db.query('SELECT * FROM articles WHERE article_id = $1', [id]).then(({rows}) => {
+    const query = `
+    SELECT 
+    articles.*,
+    COUNT(comments.comment_id) AS comment_count
+    FROM articles 
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    WHERE articles.article_id = $1
+    GROUP BY articles.article_id;
+    `;
+
```

Comment:
Nice SQL :)

Reply:
Have you updated your tests to include a comment count?

Reply:
yes i think so check article.test.js